apps.md

```diff
@@ -77,4 +77,21 @@ TimeTrapp supports:
 
 * * *
 
+## Kimai-widget - Rainmeter skin <i class="fa fa-windows"></i>
+
+Rainmeter skin for Kimai, you can start and stop the timer for a selected project and activity. Requires Rainmeter, a desktop customization tool for Windows, download and more info: [rainmeter.net](https://www.rainmeter.net/)
+
+[Download](https://github.com/infeeeee/kimai-widget/releases/latest)
+[Visit Github](https://github.com/infeeeee/kimai-widget)
+
+* * *
+
+## Kimai-cmd <i class="fa fa-windows"></i> <i class="fa fa-linux"></i> <i class="fa fa-apple"></i>
+
+A command line client for Kimai. Start and stop a timer from the terminal!
```

Comment:
While you are on it, I am excited as well that you built these apps 👍 but a simple dot "." instead of the exclamation mark "!" could be sufficient as well ;-)